lib/core_ext/active_record/connection_adapters/postgresql/schema_statements.rb

```diff
@@ -225,7 +225,7 @@ def index_name(table_name, options) #:nodoc:
         end
 
         # Override super method to provide support for expression column names.
-        def quoted_columns_for_index(column_names, **options)
+        def quoted_columns_for_index(column_names, options = {})
```

Comment:
Corresponding method in:
- Rails 6.0.0: [`ActiveRecord::ConnectionAdapters::SchemaStatements#quoted_columns_for_index`](https://github.com/rails/rails/blob/v6.0.0/activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb#L1262)
  - Note that the method signature has a double splat. In this case, that is fine because this updates to the more forgiving non-splat method signature
- Rails 6.1.0: [`ActiveRecord::ConnectionAdapters::SchemaStatements#quoted_columns_for_index`](https://github.com/rails/rails/blob/v6.1.0/activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb#L1320)